tests/test_maskedImageIO.py

```diff
@@ -39,7 +39,7 @@
 
 from builtins import object
 import numpy as np
-import pyfits
+import astropy.io
```

Comment:
Many people do this as `import astropy.io.fits as pyfits` to minimize code impact but that is probably more confusing in the long term since it leaves the `pyfits` string in the code.

Reply:
Agreed. I considered `from astropy.io import fits` but felt it was too ambiguous, and in at least one case `fits` was already used as a variable.

Reply:
Looking at other AFW code this should be plain `import astropy.io.fits` (we don't want to import all the other astropy IO plugins).